docs/docs/reference/channels.md

```diff
@@ -0,0 +1,15 @@
+# Channels
+
+::: langgraph.channels.base
```

Comment:
Maybe worth a Module-level docstring saying this section is really primarily for internal or advanced usage?

cc @nfcampos e.g., do we treat changes to these APIs as semver breaking changes?